generator-scrivito/generators/obj/index.js

```diff
@@ -0,0 +1,98 @@
+const Generator = require("yeoman-generator");
+
+module.exports = class extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+    this.log("Welcome in scrivito Obj/Widgets generator!");
+  }
+
+  start() {
+    this._creatingObj();
+  }
+
+  _creatingName(name) {
+    const defaultName = name;
+    return defaultName;
+  }
+
+  _creatingUpperName(name) {
+    const defNameUpper = name.charAt(0).toUpperCase() + name.slice(1);
+    return defNameUpper;
```

Comment:
This looks like a utility function, which is also used between obj and widget.

I suggest to create a file in `generator-scrivito/utils/capitalizeFirstLetter.js`.

```js
function capitalizeFirstLetter(string) {
  return string.charAt(0).toUpperCase() + string.slice(1);
}

module.exports = capitalizeFirstLetter;
```

It can then be used like this:
```js
const capitalizeFirstLetter = require("../../utils/capitalizeFirstLetter");
```